package.json

```diff
@@ -1,27 +1,27 @@
 {
-	"name": "metus-ingest",
-	"version": "1.2.2",
-	"api_version": "1.0.0",
-	"keywords": [
-		"Protocol",
-		"Software"
-	],
-	"manufacturer": "Metus Technology",
-	"product": "Ingest",
-	"shortname": "Metus Ingest",
-	"description": "Ingest plugin for Companion",
-	"main": "index.js",
-	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1"
-	},
-	"author": "Jeffrey Davidsz <jeffrey.davidsz@vicreo.eu>",
-	"license": "MIT",
-	"homepage": "https://github.com/bitfocus/companion-module-metus-ingest#readme",
-	"bugs": {
-		"url": "https://github.com/bitfocus/companion-module-metus-ingest/issues"
-	},
-	"repository": {
-		"type": "git",
-		"url": "git+https://github.com/bitfocus/companion-module-metus-ingest.git"
-	}
+  "name": "ingest-controller",
+  "version": "2.0.0",
+  "main": "main.js",
+  "scripts": {
+    "format": "prettier -w .",
+    "package": "companion-module-build"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bitfocus/companion-module-metus-technology-ingest-controller.git"
```

Comment:
This URL isn't correct, they 404. I don't know if there's a plan to rename it to match or not, but its currently wrong.